README.md

```diff
@@ -22,7 +22,43 @@ kubectl rook-ceph ceph status
 
 ## Commands
 
-TBD
+1. We support running Ceph command with `rook-ceph` [krew]([Krew](https://github.com/kubernetes-sigs/krew)) plugin
```

Comment:
How about removing lines 25-36? They seem redundant with the other section.

Reply:
removed.